src/env_loader.zig

```diff
@@ -70,13 +70,22 @@ pub const Lexer = struct {
             switch (variable.value[i]) {
                 '$' => {
                     i += 1;
-                    const start = i;
+                    var start: usize = i;
+
+                    const with_curly_braces = (variable.value[i] == '{');
+                    if (with_curly_braces) i += 1;
```

Comment:
It will be faster if we remove this branch
```suggestion
                    i += @as(usize, @boolToInt(with_curly_braces));
```

Reply:
Thank you for the suggestion! Haven't tuned my mindset for micro optimization yet.
Just wondering what if we store the offset directly instead of doing type conversion twice?
Like `const curly_braces_offset = if (variable.value[i] == '{') 1 else 0;`
or `const curly_braces_offset = @as(usize, @boolToInt(variable.value[i] == '{'))` if that's more performant.